constellation-app/constellation
# Clicking on the attribute field brings up the interface to edit the value

### Description of the Change
Clicking on the attribute field brings up the interface to edit the value. Since it allows to copy values by selecting the string with dragging the mouse, a single click can be used to pop up the edit window. Edit button can be removed here if it would not confuse users.


### Alternate Designs
opening up the edit window on the double click event


### Why Should This Be In Core?
Better UX


### Benefits
Better UX


### Possible Drawbacks



### Verification Process

1. Open a graph and Attribute Editor
2. Select a node and click on the field of one of the node attributes and verify it open up the interface to edit the value
3. Verify the value can be copied by selecting the string with dragging the mouse
4. Deselect everything and verify clicking on the node attribute doesn't populate anything



### Applicable Issues
https://github.com/constellation-app/constellation/issues/1242
